doc/man3/SMIME_read_ASN1.pod

```diff
@@ -9,15 +9,20 @@ SMIME_read_ASN1_ex, SMIME_read_ASN1
 
  #include <openssl/asn1.h>
 
- ASN1_VALUE *SMIME_read_ASN1_ex(BIO *in, BIO **bcont, const ASN1_ITEM *it,
-                                ASN1_VALUE **x);
+ ASN1_VALUE *SMIME_read_ASN1_ex(BIO *in, int flags, BIO **bcont,
+                                const ASN1_ITEM *it, ASN1_VALUE **x);
  ASN1_VALUE *SMIME_read_ASN1(BIO *in, BIO **bcont, const ASN1_ITEM *it);
 
 =head1 DESCRIPTION
 
 SMIME_read_ASN1_ex() parses a message in S/MIME format.
 
-I<in> is a BIO to read the message from. I<x> can be used to optionally supply
+I<in> is a BIO to read the message from.
+If the I<flags> argument contains B<CMS_BINARY> the input is assumed to be in
+binary format and is not translated to canonical form.
+If in addition B<SMIME_ASCIICRLF> is set then that the binary input is assumed
```

Comment:
then the

Reply:
Changed

Reply:
It doesnt look changed here..

Reply:
Ah, because you referred also to the line 21 I thought that you wanted to have "the input" be replaced by "then the input", which I did.
Now I see that in line 23 there is "then that the", which should be replaced by "then the" - done now.